testdata/project-v2/main.go

```diff
@@ -27,6 +27,7 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 	crewv1 "sigs.k8s.io/kubebuilder/testdata/project-v2/api/v1"
+	v1 "sigs.k8s.io/kubebuilder/testdata/project-v2/api/v1"
```

Comment:
>	crewv1 "sigs.k8s.io/kubebuilder/testdata/project-v2/api/v1"
	v1 "sigs.k8s.io/kubebuilder/testdata/project-v2/api/v1"

These 2 are actually the same import, but get duplicated.
I guess it is caused or triggered by https://github.com/kubernetes-sigs/kubebuilder/pull/1049/files#diff-c5b7fb8b471ae519bdfd5005565af33bR60-R77

I suspect `GroupImportSafe` is empty somehow: https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/scaffold/v2/main.go#L64-L77

Reply:
Hi @mengqiy, 

Thank you for get it. I tracked and issue for we check it here; https://github.com/kubernetes-sigs/kubebuilder/issues/1069